src/meshapi_hooks/models.py

```diff
@@ -1,5 +1,31 @@
 from django.db import models
+from django import forms
 
 # drf-hooks looks in this file for hook objects, but it feels weird to inline
 # it here, so we import it instead
 from .hooks import CelerySerializerHook
+class ContactPreference(models.TextChoices):
+    """Defines the available contact preference options."""
+    PHONE = 'phone', ('Phone')
+    EMAIL = 'email', ('Email')
+
+class Member(models.Model):
```

Comment:
I think you meant to modify the existing `Member` object which can be found [here](https://github.com/nycmeshnet/meshdb/blob/main/src/meshapi/models/member.py#L8)